src/menu/StudentMenu.js

```diff
@@ -10,11 +11,14 @@ export const StudentMenu = () => {
   const notifyNotImplemented = () => notify('En cours de développement. Ce qui présage quelques exercices pour vous 😉', { type: 'warning' })
   const whoamiId = authProvider.getCachedWhoami().id
   return (
-    <MultiLevelMenu variant='categories'>
-      <MenuItemCategory to='/profile' name='profile' label='Mon profil' icon={<AccountCircle />} />
-      <MenuItemCategory to={whoamiId ? `/students/${authProvider.getCachedWhoami().id}/fees` : '/'} name='fees' label='Frais' icon={<AttachMoney />} />
-      <MenuItemCategory to='/' name='student-grades' label='Notes' icon={<Receipt />} onClick={notifyNotImplemented} />
-    </MultiLevelMenu>
+    // TODO: move 'height' sx in haTheme
+    <Box sx={{ flex: 1 }}>
+      <MultiLevelMenu variant='categories' sx={{ width: '200px', backgroundColor: '#2c393a', color: 'white' }}>
+        <MenuItemCategory to='/profile' name='profile' label='Mon profil' icon={<AccountCircle />} />
+        <MenuItemCategory to={whoamiId ? `/students/${authProvider.getCachedWhoami().id}/fees` : '/'} name='fees' label='Frais' icon={<AttachMoney />} />
```

Comment:
@Fanirykeziah  do not forget to refactor this one 
use whoamiId instead of re-retrieving the cachedWhoami.id